packages/office-addin-sso/src/server.ts

```diff
@@ -5,7 +5,7 @@
 import * as https from "https";
 import * as devCerts from "office-addin-dev-certs";
 import { OfficeAddinManifest } from "office-addin-manifest";
-import { App } from "./app";
+import { App } from "./middle-tier/app";
```

Comment:
Just FYI after discussing with Ricky I am moving the term from "middle-tier" back to server, web application, or similar. I'll be updating docs appropriately (in case you are trying to map to docs.)